src/platform/datapath_raw_socket.c

```diff
@@ -226,8 +235,10 @@ CxPlatDpRawParseIPv4(
     Packet->Tuple->LocalAddress.Ipv4.sin_family = AF_INET;
     CxPlatCopyMemory(&Packet->Tuple->LocalAddress.Ipv4.sin_addr, IP->Destination, sizeof(IP->Destination));
 
-    if (IP->Protocol == 17) {
+    if (IP->Protocol == IPPROTO_UDP) {
         CxPlatDpRawParseUdp(Datapath, Packet, (UDP_HEADER*)IP->Data, Length);
+    } else {
+        QuicTraceEvent(LibraryError, "[ lib] ERROR, %s.", "unacceptable transport protocol");
```

Comment:
We follow a very explicit, 1 arg per line, format for all events in MsQuic. Please update these to match that pattern. Also, also please run `.\scripts\update-sidecar.ps1` (btw, it takes a long time).

Reply:
Also, all these "unacceptable" error cases should use `LibraryErrorStatus` and pass the value in for the "status" field, probably. It might be worth having a separate event for these too; because they might be super noisy.

Reply:
Is the 1-arg-per-line convention required by clog? Just curious why this convention is for only one type of macro call.

Reply:
It's not a hard requirement, just our common practice that I'd prefer to keep consistent.